keras/backend/cntk_backend.py

```diff
@@ -520,10 +520,10 @@ def ones(shape, dtype=None, name=None):
     return variable(value=np.ones(shape, ctype), dtype=dtype, name=name)
 
 
-def eye(size, dtype=None, name=None):
+def eye(N, M=None, dtype=None, name=None):
```

Comment:
Let's keep the signature `eye(size, ...)`

But then let's accept either an int or a tuple:

```python
if isinstance(size, (list, tuple)):
   n, m = size
else:
   n, m = size, size
```

Reply:
@fchollet, The signature has been reverted.